08-projects.Rmd

```diff
@@ -10110,7 +10100,7 @@ map <- get_map(location="United States", zoom=3)
 ggmap(map)
 ```
 
-**Hint:** See [here](https://stackoverflow.com/questions/23130604/plot-coordinates-on-map) for an example of adding points to a map.
+**Hint:** See [here](#how-do-i-add-latitude-and-longitude-points-to-a-map-of-purdue-university) for an example of adding points to a map.
```

Comment:
Hi @kevinamstutz , while I have not tried this example, it looks quite relevant to this question. However, if you were using the Stack Overflow link for other reasons, please feel free to keep the original link.

Reply:
I agree, I'm not sure why. I probably added the example after I wrote the question and never changed it. Thanks!